pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -111,7 +111,9 @@ class ModelCheckpoint(Callback):
 
     """
 
-    CHECKPOINT_NAME_LAST = "last.ckpt"
+    CHECKPOINT_JOIN_CHAR = "-"
+    CHECKPOINT_NAME_LAST = "last"
+    CHECKPOINT_SUFFIX = ".ckpt"
```

Comment:
Can we do something like, if `suffix` is present in `filepath`, then keep it as it is else do `.ckpt`. This way we could avoid `CHECKPOINT_SUFFIX`. Also this should be `CHECKPOINT_NAME_LAST=`checkpoint_callback_last_model_path`. What do you think?

Reply:
`CHECKPOINT_SUFFIX` would not be avoided if we implement `get_suffix_from_filepath()` because we would still need a default behaviour. IMO it is not worth it because we probably want all users to use the same suffix (`ckpt`) for consistency. I just extracted `.ckpt` into a variable to have it in one single place.

Do you mean adding `CHECKPOINT_STATE_LAST_PATH=checkpoint_callback_last_model_path`?

Reply:
Then I don't think it requires a variable here just to specify the default suffix.

Reply:
Okay then, I will leave the value hard-coded.